src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -533,7 +533,8 @@ void FastRouteCore::initBlockedIntervals(std::vector<int>& track_space)
     if (edge_cap > 0) {
       int reduce = 0;
       for (auto interval_it : intervals) {
-        reduce += ceil(std::abs(interval_it.upper() - interval_it.lower())
+        reduce += ceil(static_cast<float>(
+                           std::abs(interval_it.upper() - interval_it.lower()))
                        / track_space[layer - 1]);
```

Comment:
@luis201420 why is "track_space[layer - 1]" the divisor and not "track_space[layer]"?  It looks like we are using the wrong layer.

Reply:
Never mind, I got tripped up by dbTechLayer::getRoutingLevel() being 1-based which is where the -1 comes from.